tests/ParserTest.elm

```diff
@@ -558,53 +573,132 @@ expr =
 
                 -- https://github.com/elm/compiler/blob/dcbe51fa22879f83b5d94642e117440cb5249bb1/compiler/src/Parse/String.hs#L279-L285
                 , ( "escape n"
-                  , "'\\n'"
+                  , singleQuote "\\n"
                   , Ok (Literal (Char '\n'))
                   )
                 , ( "escape r"
-                  , "'\\r'"
+                  , singleQuote "\\r"
                   , Ok (Literal (Char '\u{000D}'))
                   )
                 , ( "escape t"
-                  , "'\\t'"
+                  , singleQuote "\\t"
                   , Ok (Literal (Char '\t'))
                   )
                 , ( "double quote"
-                  , "'\\\"'"
+                  , singleQuote "\\\""
                   , Ok (Literal (Char '"'))
                     -- " (for vscode-elm bug)
                   )
                 , ( "single quote"
-                  , "'\\''"
+                  , singleQuote "'\\''"
```

Comment:
Is this correct? I'd expect `singleQuote "'"`

Reply:
Fixed by https://github.com/elm-in-elm/compiler/pull/31/commits/51d0b7fbad7f45642e392142f5acad0f1d6ad1cc